Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -66,14 +66,13 @@ open class ChatChannelListVC: _ViewController,
     /// - Returns: A newly created ChatChannelListViewController
     public static func createChannelVC(
         _ controller: ChatChannelListController,
-        storyboard: UIStoryboard?,
+        storyboard: UIStoryboard? = nil,
         storyboardId: String? = nil
     ) -> ChatChannelListVC {
         var chatChannelListVC: ChatChannelListVC!
         
-        if let storyboardId = storyboardId {
-            let storyboard = storyboard
-            chatChannelListVC = storyboard?.instantiateViewController(withIdentifier: storyboardId) as? ChatChannelListVC
+        if let storyboardId = storyboardId, let storyboard = storyboard {
+            chatChannelListVC = storyboard.instantiateViewController(withIdentifier: storyboardId) as? ChatChannelListVC
```

Comment:
We still have an unhandled case here. If the chatChannelListVC is nil, it will crash in the setup lines below, since it's implicitly unwrapped above. Question is should we allow it to crash like that, or throw a fatal error with explanation, since most likely that would be a developer's error?